docs/atlascli/command/atlas-backups-compliancePolicy-policies-ondemand-update.txt

```diff
@@ -0,0 +1,85 @@
+.. _atlas-backups-compliancePolicy-policies-ondemand-update:
+
+=======================================================
+atlas backups compliancePolicy policies ondemand update
+=======================================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Update the on-demand policy of the backup compliance for your project.
+
+Syntax
+------
+
+.. code-block::
+   :caption: Command Syntax
+
+   atlas backups compliancePolicy policies ondemand update [options]
+
+.. Code end marker, please don't delete this comment
+
+Options
+-------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - -h, --help
+     - 
+     - false
+     - help for update
+   * - -o, --output
+     - string
+     - false
+     - Output format. Valid values are json, json-path, go-template, or go-template-file. To see full output, use the -o json option.
+   * - --projectId
+     - string
+     - false
+     - Hexadecimal string that identifies the project to use. This option overrides the settings in the configuration file or environment variable.
+   * - --retentionUnit
+     - string
+     - true
+     - Unit of time in which MongoDB Cloud measures snapshot retention: 'days' 'weeks' 'months'. 
+   * - --retentionValue
+     - int
+     - true
+     - Duration in days, weeks, or months that MongoDB Cloud retains the snapshot. For less frequent policy items, MongoDB Cloud requires that you specify a value greater than or equal to the value specified for more frequent policy items.
```

Comment:
```suggestion
     - Duration in days, weeks, or months that Atlas retains the snapshot. For less frequent policy items, Atlas requires that you specify a value greater than or equal to the value specified for more frequent policy items.
```
Could you please replace MongoDB Cloud with Atlas throughout this PR?

Reply:
yep, updated! thanks